runtime/win32.c

```diff
@@ -816,23 +816,8 @@ int caml_win32_rename(const wchar_t * oldpath, const wchar_t * newpath)
                  MOVEFILE_COPY_ALLOWED)) {
     return 0;
   }
-  /* Modest attempt at mapping Win32 error codes to POSIX error codes.
-     The __dosmaperr() function from the CRT does a better job but is
-     generally not accessible. */
-  switch (GetLastError()) {
-  case ERROR_FILE_NOT_FOUND: case ERROR_PATH_NOT_FOUND:
-    errno = ENOENT; break;
-  case ERROR_ACCESS_DENIED: case ERROR_WRITE_PROTECT: case ERROR_CANNOT_MAKE:
-    errno = EACCES; break;
-  case ERROR_CURRENT_DIRECTORY: case ERROR_BUSY:
-    errno = EBUSY; break;
```

Comment:
The new code maps `ERROR_CURRENT_DIRECTORY` to `EACCES`, not `EBUSY`, for consistency with `_dosmaperr` and the Win32 Unix library.  It's a small change to map it back to `EBUSY`, but I'm not sure it's worth departing from `_dosmaperr` here.